modules/module-mysql/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@powersync/service-module-mysql",
+  "repository": "https://github.com/powersync-ja/powersync-service",
+  "types": "dist/index.d.ts",
+  "version": "0.0.1",
+  "license": "FSL-1.1-Apache-2.0",
+  "main": "dist/index.js",
+  "type": "module",
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "build": "tsc -b",
+    "build:tests": "tsc -b test/tsconfig.json",
+    "clean": "rm -rf ./dist && tsc -b --clean",
+    "test": "vitest"
+  },
+  "exports": {
+    ".": {
+      "import": "./dist/index.js",
+      "require": "./dist/index.js",
+      "default": "./dist/index.js"
+    },
+    "./types": {
+      "import": "./dist/types/types.js",
+      "require": "./dist/types/types.js",
+      "default": "./dist/types/types.js"
+    }
+  },
+  "dependencies": {
+    "@powersync/lib-services-framework": "workspace:*",
+    "@powersync/service-core": "workspace:*",
+    "@powersync/service-sync-rules": "workspace:*",
+    "@powersync/service-types": "workspace:*",
+    "@powersync/service-jsonbig": "workspace:*",
+    "@powersync/mysql-zongji": "0.0.0-dev-20241031142605",
```

Comment:
Wondering if we should go ahead an create a Zongji release that we can use here.

Reply:
Bumped to 0.1.0 in https://github.com/powersync-ja/powersync-service/pull/137